### DISCUSSION TITLE
Populate Theme Name field when selecting Theme

### DISCUSSION THREAD

**Discussion by cbaxter7** [vueform/builder#82] [Questions] [answered]
Upon selecting a theme via the 'Load Theme' dropdown, I've noticed that the 'Name' field is not populated with the selected theme name.

**Comment by adamberecz** [comment]
That's because the "official" list of themes provided by the builder should not be overwritten. If you load one that is saved by the user then the name should be loaded so that it gets saved under that name if the user hits Save. Otherwise they would have to provide it a custom name.

**Reply by cbaxter7** [reply]
In our case we aren't looking at the "official" list of themes, we are passing in our own set of themes into the builder config.

Another potential question is could we possibly update the theme when we select from the list rather than click load and then retain the selection within the dropdown? Or if we do click load, retain the selection within the dropdown?

**Reply by adamberecz** [reply, marked as answer]
By the "official" list I mean the list of themes that are provided to the users by the developer of the platform. Those aren't editable by the users.

It's not possible out-of-the-box, but something could be managed with a custom field. Look for `fields/ThemeThemeField.js` and make a copy of it for further customization.
